app/Models/Sensor.php

```diff
@@ -98,30 +98,34 @@ public static function getIconMap()
         return self::$icons;
     }
 
-    public function guessLimits(): void
+    public function guessLimits(bool $high, bool $low): void
     {
-        $this->sensor_limit_low = match ($this->sensor_class) {
-            'temperature' => $this->sensor_current - 10,
-            'voltage' => $this->sensor_current * 0.85,
-            'humidity' => 30,
-            'fanspeed' => $this->sensor_current * 0.80,
-            'power_factor' => -1,
-            'signal' => -80,
-            'airflow', 'snr', 'frequency', 'pressure', 'cooling' => $this->sensor_current * 0.95,
-            default => null,
-        };
-
-        $this->sensor_limit = match ($this->sensor_class) {
-            'temperature' => $this->sensor_current + 20,
-            'voltage' => $this->sensor_current * 1.15,
-            'humidity' => 70,
-            'fanspeed' => $this->sensor_current * 1.80,
-            'power_factor' => 1,
-            'signal' => -30,
-            'load' => 80,
-            'airflow', 'snr', 'frequency', 'pressure', 'cooling' => $this->sensor_current * 1.05,
-            default => null,
-        };
+        if ($low) {
```

Comment:
The current broken test. Wondering if low/high is flipped wrong or if the test data is wrong maybe

Reply:
So, the legacy code had this thing where it flips high and low if one is higher than the other. The new code changes WHEN that happens...
But, to me this test looks like the 0s should be null (unset) which would prevent the swap.  I was trying to figure out the correct setting, but my sleepy brain couldn't reason through it :)